CHANGELOG.md

```
@@ -14,6 +14,17 @@ Description.
 **Full Changelog**: https://github.com/bdmendes/smockito/compare/<prev>...<this>
 -->
 
+## 2.5.1 - 2026-01-31
+
+This release fixes an edge case related to the autoboxing of primitives and improves error messages.
+
+### What's Changed
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)